doc/submitter_guide.md

```diff
@@ -95,11 +96,10 @@ exit status.
 3\. Generate some report data
 -----------------------------
 
-KCIDB accepts data in JSON, described by a versioned schema. The
-`kcidb-schema` tool will output the current schema version. However, all tools
-will accept data complying with older schema versions, as a rule. Any
-exceptions to that will be mentioned in KCIDB release notes. Pipe your data
-into `kcidb-validate` tool to check if it will be accepted.
+`kcidb-schema` tool will output the current schema version. However, all tools will accept data complying with older schema versions. 
```

Comment:
Please wrap this paragraph to 78 characters.

Reply:
Done